test/nodes/test_query_classifier.py

```diff
@@ -0,0 +1,51 @@
+from haystack.nodes.query_classifier.base import BaseQueryClassifier
+
+
+def test_transformers_query_classifier(transformers_query_classifier):
+    assert isinstance(transformers_query_classifier, BaseQueryClassifier)
```

Comment:
I don't think we need this assert (valid also for the other tests). That would be testing the test code :smile:

Reply:
I copied from other tests :smile:. I'll remove it.